### DISCUSSION TITLE
GoPhish - Donation Inquiry

### DISCUSSION THREAD

**Discussion by mikegvalerio** [gophish/gophish#2593] [General]
Hello, 

I just wanted to see if there was any information where monetary donations (for the platform) can be sent to?

**Comment by mikegvalerio** [comment]
Hello,

I just wanted to follow up and see if there were any updates here?

**Comment by glennzw** [comment]
Hi @jarvismgv - thanks for reaching out, and that's a very kind offer. Drop me an email if you'd like to discuss this further.

**Comment by glennzw** [comment]
Hi @jarvismgv - I set up a github sponsor page to support my maintenance of the project:

https://github.com/sponsors/glennzw

In the longer run I'll figure out a way to add a sponsor option to the project itself (tax plumbing and all that is the tricky bit).

**Reply by mikegvalerio** [reply]
Hey @glennzw,

Thanks for sharing this!
